src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -200,6 +201,66 @@ public virtual LocalBuilder DeclareLocal(Type localType)
         [CLSCompliant(false)]
         public void Emit(OpCode opcode, sbyte arg) => Emit(opcode, (byte)arg);
 
+        /// <summary>
+        /// Marks a sequence point in the Microsoft intermediate language (MSIL) stream.
+        /// </summary>
+        /// <param name="document">The document for which the sequence point is being defined.</param>
+        /// <param name="startLine">The line where the sequence point begins.</param>
+        /// <param name="startColumn">The column in the line where the sequence point begins.</param>
+        /// <param name="endLine">The line where the sequence point ends.</param>
+        /// <param name="endColumn">The column in the line where the sequence point ends.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="document"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="document"/> is not valid.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// <paramref name="startLine"/> is not within range [0, 0x20000000) or
```

Comment:
Where did these max values come from?

Reply:
From the runtime doc https://github.com/dotnet/runtime/blob/617f81f3145ad0829e38101b4abc6c98b201542a/docs/design/specs/PortablePdb-Metadata.md?plain=1#L124-L133